src/resources/js/router.js

```diff
@@ -19,6 +20,12 @@ const router = new VueRouter({
       name: 'Test',
       icon: 'mdi-help-box',
       component: Hello,
+    },
+    {
+      path: '/Authentification',
```

Comment:
the url should be `/login` or `/connexion` in this case event if you go for the english word it's acceptable in french to say login

Reply:
changed in 7adc1a7